package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "devDependencies": {
+    "jasmine-browser-runner": "^1.0.0",
+    "jasmine-core": "^4.0.1"
+  },
+  "scripts": {
+    "jasmine:prepare": "RAILS_ENV=test bundle exec rails assets:clobber assets:precompile",
+    "jasmine:ci": "yarn run jasmine:prepare && yarn run jasmine-browser-runner runSpecs",
+    "jasmine:browser": "yarn run jasmine:prepare && yarn run jasmine-browser-runner"
+  }
```

Comment:
We could use some more of the conventional package.json information in here. 

For example:

```
  "name": "transition",
  "description": "GOV.UK application to manage the transition of websites to GOV.UK",
  "private": true,
  "author": "Government Digital Service",
  "license": "MIT",
```

Reply:
It is done 👍